src/pages/jobLogs/JobLogsTable.tsx

```diff
@@ -0,0 +1,195 @@
+import { useQuery } from "@apollo/client";
+import styled from "@emotion/styled";
+import Icon from "@leafygreen-ui/icon";
+import { uiColors } from "@leafygreen-ui/palette";
+import { Table, TableHeader, Row, Cell } from "@leafygreen-ui/table";
+import { useLocation } from "react-router-dom";
+import { useJobLogsAnalytics } from "analytics/joblogs/useJobLogsAnalytics";
+import { Button } from "components/Button";
+import { size } from "constants/tokens";
+import { useToastContext } from "context/toast";
+import {
+  GetDisplayTaskQuery,
+  GetTestsQuery,
+  GetTestsQueryVariables,
+} from "gql/generated/types";
+import { GET_TESTS } from "gql/queries";
+import { JobLogsPopover } from "pages/jobLogs/JobLogsPopover";
+import { PageSizeSelect } from "pages/jobLogs/PageSizeSelect";
+import { PaginationButtons } from "pages/jobLogs/PaginationButtons";
+
+import { queryString, url } from "utils";
+
+const { parseQueryString, getString } = queryString;
+const { getPageFromSearch, getLimitFromSearch } = url;
+const { gray } = uiColors;
+
+enum JobLogsQueryParams {
+  Category = "sortBy",
+  SortDir = "sortDir",
+  Page = "page",
+  Limit = "limit",
+  Test = "test",
+}
+
+interface JobLogsTableProps {
+  task: GetDisplayTaskQuery["task"]["displayTask"];
+  groupId?: string;
+}
+
+export const JobLogsTable: React.FC<JobLogsTableProps> = ({
+  task,
+  groupId,
+}) => {
+  const dispatchToast = useToastContext();
+  const { sendEvent } = useJobLogsAnalytics();
+
+  const { search } = useLocation();
+  const queryVariables = getQueryVariables(search, task, groupId);
+  const { limitNum, pageNum } = queryVariables;
+
+  const { data: testData, loading: isLoadingTests } = useQuery<
+    GetTestsQuery,
+    GetTestsQueryVariables
+  >(GET_TESTS, {
+    variables: queryVariables,
+    fetchPolicy: "network-only",
+    nextFetchPolicy: "cache-and-network",
```

Comment:
Why change fetchPolicies?

Reply:
sorry I copy and pasted another query and forgot to change the `fetchPolicy` 🧍